neptune/new/attributes/constants.py

```diff
@@ -36,3 +36,4 @@
 SYSTEM_NAME_ATTRIBUTE_PATH = f'{SYSTEM_ATTRIBUTE_SPACE}name'
 SYSTEM_STATE_ATTRIBUTE_PATH = f'{SYSTEM_ATTRIBUTE_SPACE}state'
 SYSTEM_TAGS_ATTRIBUTE_PATH = f'{SYSTEM_ATTRIBUTE_SPACE}tags'
+SYSTEM_FAILED_ATTRIBUTE_PATH = f'{SYSTEM_ATTRIBUTE_SPACE}failed'
```

Comment:
Please rebase to https://github.com/neptune-ai/neptune-client/pull/521 where it's added

Reply:
It has not been merged yet.